python/graphstorm/model/node_glem.py

```diff
@@ -128,6 +128,21 @@ def gnn_encoder(self):
         """Alias for accessing the gnn_encoder"""
         return self.gnn.gnn_encoder
 
+    @property
+    def node_input_encoder(self):
+        """Alias for accessing the node_input_encoder"""
+        return self.lm.node_input_encoder
+
+    @property
+    def _node_input_encoder(self):
+        """Alias for accessing the node_input_encoder"""
+        return self.lm.node_input_encoder
+
+    @property
+    def decoder(self):
+        """Alias for accessing the decoder"""
+        return self.gnn.decoder if self.inference_using_gnn else self.lm.decoder
```

Comment:
you already define these. why do you still need the code above?

Reply:
They need to be accessed in other functions in `embed.py`, `do_full_graph_inference`, etc.

Reply:
i understand. what i mean is that we should redefine these properties and we don't need the modifications in np_trainer.py or np_infer.py